backend/users/src/main/java/org/eclipse/sw360/users/util/ReadFileDepartmentConfig.java

```diff
@@ -34,7 +34,7 @@ protected String getPathConfig() throws IOException {
         File file = File.createTempFile("check", "text");
         String pathFile = file.getPath();
         String[] parts = pathFile.split("/");
-        for (int i = 0; i < parts.length; i++) {
+        for (int i = 0; i < parts.length-1; i++) {
             if (!parts[i+1].contains("liferay"))
```

Comment:
Since Liferay has been removed, we can move the configuration file (department-config.json) to the Tomcat folder. Could you help me update the source code as follows?

```suggestion
            if (!parts[i+1].contains("tomcat"))
```

Reply:
@nikkuma7  I plan to move these configurations into the database in the future (instead of using json file), but for now, let's fix the above to get the functionality working